pkg/api/rest/rest.go

```diff
@@ -62,7 +62,7 @@ type Lister interface {
 	// This object must be a pointer type for use with Codec.DecodeInto([]byte, runtime.Object)
 	NewList() runtime.Object
 	// List selects resources in the storage which match to the selector.
-	List(ctx api.Context, label labels.Selector, field fields.Selector) (runtime.Object, error)
+	List(ctx api.Context, label labels.Selector, field fields.Selector, options *api.ListOptions) (runtime.Object, error)
```

Comment:
Typically this is untyped (GetterWithOptions accepts runtime.Object) and we support both the old and the new interface.  However, since List() is effectively standard and we don't have a use case for api.ListOptions varying between callers, I'm ok with making this change as is.

Can you add a TODO on this interface to remove label and field selectors (since they are part of api.ListOptions)?